R/utils_visr.R

```diff
@@ -61,6 +60,10 @@ align_plots <- function(pltlist) {
     x$widths <- max.widths
     x
   })
+  
+  # Move y-axis label back closer to the axis
+  strata_tick_width <- grid::convertUnit(plots.grobs.eq.widths[[2]]$grobs[[3]]$width, "cm")
+  plots.grobs.eq.widths[[1]]$grobs[[13]]$children[[1]]$x <- strata_tick_width - grid::unit(0.85, "cm")
```

Comment:
Here I am worried. Why 0.85? is this emperical? If yes we need a more robust solution. You can check how cowplot is doing this when setting argument axis = "l".